src/System.Reflection.Metadata/src/System/Reflection/Metadata/AssemblyReference.cs

```diff
@@ -135,13 +134,8 @@ public CustomAttributeHandleCollection GetCustomAttributes()
         #region Virtual Rows
         private Version GetVirtualVersion()
         {
-            switch ((AssemblyReferenceHandle.VirtualIndex)RowId)
-            {
-                case AssemblyReferenceHandle.VirtualIndex.System_Numerics_Vectors:
```

Comment:
Can you remove the System_Numerics_Vectors enum value now as well? Or is it needed for projections still in other locations?